Adjust for disabled-by default rseq in downstream: tst-rseq needs to enable
rseq using the tunable, tst-rseq-disable should use the default.

diff --git a/sysdeps/unix/sysv/linux/Makefile b/sysdeps/unix/sysv/linux/Makefile
index 856a9d58cef6a879..23a577e263d6dc22 100644
--- a/sysdeps/unix/sysv/linux/Makefile
+++ b/sysdeps/unix/sysv/linux/Makefile
@@ -233,7 +233,7 @@ $(objpfx)tst-mman-consts.out: ../sysdeps/unix/sysv/linux/tst-mman-consts.py
 	  < /dev/null > $@ 2>&1; $(evaluate-test)
 $(objpfx)tst-mman-consts.out: $(sysdeps-linux-python-deps)
 
-tst-rseq-disable-ENV = GLIBC_TUNABLES=glibc.pthread.rseq=0
+tst-rseq-ENV = GLIBC_TUNABLES=glibc.pthread.rseq=1
 
 endif # $(subdir) == misc
 
